weather_aplication/CustomTableViewCell.swift

```diff
@@ -27,30 +36,66 @@ class CustomTableViewCell: UITableViewCell {
         addSubview(temperatureLabel)
         addSubview(pressureLabel)
         addSubview(windSpeedLabel)
+        addSubview(weatherImageView)
     }
     
     func setupConstraints() {
         hourLabel.snp.makeConstraints { make in
             make.leftMargin.equalToSuperview()
+            make.centerY.equalToSuperview()
         }
 
         temperatureLabel.snp.makeConstraints { make in
             make.left.lessThanOrEqualTo(hourLabel.snp.right).offset(kSideMargin)
+            make.centerY.equalToSuperview()
         }
 
         pressureLabel.snp.makeConstraints { make in
             make.left.lessThanOrEqualTo(temperatureLabel.snp.right).offset(kSideMargin)
+            make.centerY.equalToSuperview()
         }
         
         windSpeedLabel.snp.makeConstraints { make in
             make.left.lessThanOrEqualTo(pressureLabel.snp.right).offset(kSideMargin)
+            make.centerY.equalToSuperview()
+        }
+        
+        weatherImageView.snp.makeConstraints { make in
+            make.left.lessThanOrEqualTo(windSpeedLabel.snp.right).offset(kSideMargin)
+            make.centerY.equalToSuperview()
         }
     }
     
-    func setupData(cellData: CellData) {
+    func setupData(cellData: WeatherData) {
         self.hourLabel.text = cellData.hour
-        self.temperatureLabel.text = "\(cellData.temperature)C"
-        self.pressureLabel.text = "\(cellData.pressure)hPa"
-        self.windSpeedLabel.text = "\(cellData.windSpeed)km/h"
+        self.temperatureLabel.text = "\(cellData.temperature)"
+        self.pressureLabel.text = "\(cellData.pressure)"
+        self.windSpeedLabel.text = "\(cellData.windSpeed)"
+        self.weatherImageView.image = choseImage(weatherCode: cellData.weatherCode)
+    }
+    
+    func choseImage(weatherCode: Int) -> UIImage? {
+        switch weatherCode {
```

Comment:
ładnie to wygląda, tylko wszystkie elementy cellki są przesunięte do lewej strony. Możesz poczytać sobie o czymś takim jak UIStackView i właśnie z niego skorzystać żeby każda labelki cellki były równomiernie rozmieszczone